math/genvector/inc/Math/GenVector/DisplacementVector3D.h

```diff
@@ -337,18 +337,18 @@ namespace ROOT {
       /**
          return unit vector parallel to this (scalar)
       */
-      template <typename SCALAR = Scalar>
-      typename std::enable_if<std::is_arithmetic<SCALAR>::value, DisplacementVector3D>::type Unit() const
+      template <typename SCALAR = Scalar, typename std::enable_if<std::is_arithmetic<SCALAR>::value>::type * = nullptr>
```

Comment:
Actually, it fails to compile when there are two methods. i.e.

```    
 /**
         return unit vector parallel to this (scalar)
      */
      template <typename SCALAR = Scalar, typename = typename std::enable_if<std::is_arithmetic<SCALAR>::value>::type>
      DisplacementVector3D Unit() const
      {
         const auto tot = R();
         return tot == 0 ? *this : DisplacementVector3D(*this) / tot;
      }

      /**
         return unit vector parallel to this (vector)
      */
      template <typename SCALAR = Scalar, typename = typename std::enable_if<!std::is_arithmetic<SCALAR>::value>::type>
      DisplacementVector3D Unit() const
      {
         SCALAR tot            = R();
         tot(tot == SCALAR(0)) = SCALAR(1);
         return DisplacementVector3D(*this) / tot;
      }
```

gives

```
In file included from /Users/chris/Projects/ROOT/source/math/genvector/src/Boost.cxx:16:
In file included from /Users/chris/Projects/ROOT/build/include/Math/GenVector/Boost.h:20:
In file included from /Users/chris/Projects/ROOT/build/include/Math/GenVector/LorentzVector.h:23:
/Users/chris/Projects/ROOT/build/include/Math/GenVector/DisplacementVector3D.h:351:28: error: class member cannot be redeclared
      DisplacementVector3D Unit() const
                           ^
/Users/chris/Projects/ROOT/build/include/Math/GenVector/DisplacementVector3D.h:341:28: note: previous declaration is here
      DisplacementVector3D Unit() const
                           ^
```

I'll stick with what is currently there and works.